lib/stm32/l0/Makefile

```diff
@@ -35,15 +35,17 @@ TGT_CFLAGS      += $(DEBUG_FLAGS)
 
 ARFLAGS		= rcs
 
-OBJS		= gpio.o rcc.o desig.o
+OBJS		= flash.o gpio.o rcc.o desig.o
 OBJS		+= pwr_common_all.o pwr_common_l01.o
 
-OBJS            += gpio_common_all.o gpio_common_f0234.o rcc_common_all.o
+OBJS		+= gpio_common_all.o gpio_common_f0234.o rcc_common_all.o
 OBJS		+= adc_common_v2.o
-OBJS		+= crs_common_all.o
+OBJS		+= crs_common_all.o flash_common_l01.o
+OBJS		+= i2c_common_l0f03.o usart_common_l0f0.o
+OBJS		+= usart.o usart_common_v2.o
 
-OBJS            += usb.o usb_control.o usb_standard.o
-OBJS            += st_usbfs_core.o st_usbfs_v2.o
+OBJS		+= usb.o usb_control.o usb_standard.o
+OBJS		+= st_usbfs_core.o st_usbfs_v2.o
```

Comment:
While you've probably "fixed" things here, please avoid making unrelated whitespace changes in the same commit.  It makes merges harder.